test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || (atlas && kubernetes)
```

Comment:
[not blocking] we add the cluster tag when the cluster is needed for the e2e test. In your case, you are creating a project and a cluster
```suggestion
//go:build e2e || (atlas && cluster&& kubernetes)
```